kolibri/content/api.py

```diff
@@ -82,43 +82,7 @@ def leaves(channel_id=None, content=None, **kwargs):
     return content.get_leafnodes().using(channel_id)
 
 @can_get_content_with_id
-def get_all_formats(channel_id=None, content=None, **kwargs):
-    """
-    Get all possible formats for a particular content including its descendants' formats.
-
-    :param channel_id: str
-    :param content: ContentMetadata or str
-    :return: QuerySet of Format
-    """
-    all_end_nodes = leaves(channel_id=channel_id, content=content)
-    return KolibriContent.Format.objects.using(channel_id).filter(contentmetadata__in=all_end_nodes)
-
-@can_get_content_with_id
-def get_available_formats(channel_id=None, content=None, **kwargs):
-    """
-    Get all available formats for a particular content excluding its descendants' formats.
-    if the pass-in content is a topic, this function will return null.
-
-    :param channel_id: str
-    :param content: ContentMetadata or str
-    :return: QuerySet of Format
-    """
-    return KolibriContent.Format.objects.using(channel_id).filter(contentmetadata=content, available=True)
-
-@can_get_content_with_id
-def get_possible_formats(channel_id=None, content=None, **kwargs):
-    """
-    Get all possible formats for a particular content excluding its descendants' formats.
-    if the pass-in content is a topic, this function will return null.
-
-    :param channel_id: str
-    :param content: ContentMetadata or str
-    :return: QuerySet of Format
-    """
-    return KolibriContent.Format.objects.using(channel_id).filter(contentmetadata=content)
-
-@can_get_content_with_id
-def get_files_for_quality(channel_id=None, content=None, format_quality=None, **kwargs):
+def get_files_for_quality(channel_id=None, content=None, preset_quality=None, **kwargs):
```

Comment:
The word "quality" doesn't seem general enough anymore. Presets are not just about quality.